internal/runner/service.go

```diff
@@ -648,7 +648,7 @@ func readLoop(
 func runnerWinsizeToPty(winsize *runnerv1.Winsize) *pty.Winsize {
 	if winsize == nil {
 		// sane default
-		return &pty.Winsize{Cols: 80}
+		return &pty.Winsize{Rows: 24, Cols: 80}
```

Comment:
It helps to resolve flaky service tests on Linux. 🤷🏻‍♂️

Reply:
Definitely a good call to have it set even if didn't affect the tests.